packages/icondusk-e2e/package.py

```diff
@@ -29,12 +31,17 @@ def cmake_args(self):
         args = []
         spec = self.spec
 
+
+        gcclibdir = os.path.dirname(subprocess.run([self.compiler.cxx,"-print-file-name=libstdc++.so"], capture_output=True).stdout.decode("utf-8"))
         args.append('-DCMAKE_BUILD_TYPE={0}'.format(self.spec.variants['build_type'].value))
         args.append('-DPython3_EXECUTABLE=' + spec['python'].prefix +'/bin/python3.8')
         args.append('-Datlas_utils_ROOT='+spec['atlas_utilities'].prefix)
         args.append('-Ddawn4py_DIR='+spec['dawn4py'].prefix)
         args.append('-Datlas_DIR='+spec['atlas'].prefix)
         args.append('-DPRECISION='+spec.variants['precision'].value)
+        # Hack in order to fix this spack issue with RPATH
+        # https://github.com/spack/spack/issues/4261#issuecomment-466005631
+        args.append('-DCMAKE_INSTALL_RPATH='+gcclibdir)
```

Comment:
do you understand why don't we have this issue with the gt dycore and cosmo ?

Reply:
it is triggered by some dependencies, but I dont understand the mechanism or what exactly triggers this extra_rpaths thing. 
However one clear difference here is that this package depends on python3.8
Since it is not in the tsa system, it will actually compile python3.8, which triggers ssl and many other system dependencies. 
This does not happen in COSMO, where all the system dependencies are pretty much installed by modules